.github/workflows/backend-rollback.yml

```
@@ -0,0 +1,120 @@
+          script: |
+            await github.rest.repos.createDeploymentStatus({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              deployment_id: ${{ steps.create_deployment.outputs.deployment_id }},
+              state: 'failure',
+              description: 'Rollback failed'
+            });
+
+      - name: Rollback Summary
+        if: always()
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:1:41: Double quote to prevent globbing and word splitting

(shellcheck)